eq-author-api/schema/resolvers/base.js

```diff
@@ -1076,18 +1071,9 @@ const Resolvers = {
 
       return find(pages, { id: pageId });
     },
-    positive: ({ positiveLabel, positiveDescription }) => ({
-      label: positiveLabel,
-      description: positiveDescription,
-    }),
-    negative: ({ negativeLabel, negativeDescription }) => ({
-      label: negativeLabel,
-      description: negativeDescription,
-    }),
     availablePipingAnswers: ({ id }, args, ctx) =>
-      ctx.repositories.QuestionConfirmation.getPipingAnswers(id),
-    availablePipingMetadata: ({ id }, args, ctx) =>
-      ctx.repositories.QuestionConfirmation.getPipingMetadata(id),
+      getPreviousAnswersForPage(ctx.questionnaire, id),
+    availablePipingMetadata: (page, args, ctx) => ctx.questionnaire.metadata,
```

Comment:
Are these filtered by type? Or can we pipe all metadata?

Reply:
You can pipe any metadata into a question confirmation.